test/definitions/native.jl

```diff
@@ -114,12 +114,12 @@ module LazyCodegen
             @assert !cc.compiled
             job = cc.job
 
-            entry_name, jitted_mod = JuliaContext() do ctx
+            name, jitted_mod = JuliaContext() do ctx
```

Comment:
FWIW, this is wrong. By assigning to `name`, the call to `LLVM.name` in the block is broken.

Reply:
Odd that this wasn't caught in CI :thinking: But yeah, shows that a quick `sed` is often more dangerous than one might think :grimacing:

Reply:
It was caught, but resulted in a timeout which we ignored...

Reply:
Ah :| Mea culpa then! Still surprising that the others ran through, but it is what it is. Good thing you caught & fixed it :+1: